package.json

```diff
@@ -43,40 +43,40 @@
     "vue-affix": "^0.5.2",
     "vue-router": "^3.4.3",
     "vue2-filters": "^0.10.0",
-    "vuelidate": "^0.7.5",
-    "vuetify": "^2.3.9",
+    "vuelidate": "^0.7.7",
+    "vuetify": "^2.3.15",
     "vuex": "^3.5.1",
     "webpack-remove-serviceworker-plugin": "^1.0.0"
   },
   "devDependencies": {
     "@types/jest": "^25.2.3",
-    "@types/lodash": "^4.14.159",
-    "@types/vuelidate": "^0.7.13",
+    "@types/lodash": "^4.14.185",
+    "@types/vuelidate": "^0.7.15",
     "@typescript-eslint/eslint-plugin": "^2.34.0",
     "@typescript-eslint/parser": "^2.34.0",
-    "@vue/cli-plugin-babel": "4.4.1",
-    "@vue/cli-plugin-eslint": "4.2.3",
-    "@vue/cli-plugin-typescript": "4.2.3",
-    "@vue/cli-plugin-unit-jest": "4.2.3",
-    "@vue/cli-service": "4.2.3",
+    "@vue/cli-plugin-babel": "^4.5.19",
+    "@vue/cli-plugin-eslint": "^4.5.19",
+    "@vue/cli-plugin-typescript": "^4.5.19",
+    "@vue/cli-plugin-unit-jest": "^4.5.19",
+    "@vue/cli-service": "^4.5.19",
     "@vue/eslint-config-standard": "^4.0.0",
     "@vue/eslint-config-typescript": "^4.0.0",
     "@vue/test-utils": "1.0.0-beta.28",
     "eslint": "^6.8.0",
     "eslint-plugin-vue": "^6.2.2",
     "flush-promises": "^1.0.2",
-    "sass": "^1.26.10",
+    "sass": "~1.32.13",
     "sass-loader": "^8.0.2",
     "sinon": "^9.0.3",
     "typescript": "^3.9.7",
-    "vue-class-component": "^7.2.5",
-    "vue-cli-plugin-vuetify": "^2.0.7",
+    "vue-class-component": "^7.2.6",
+    "vue-cli-plugin-vuetify": "^2.5.8",
     "vue-debounce-decorator": "^1.0.1",
     "vue-plugin-helper-decorator": "^0.0.12",
-    "vue-property-decorator": "^8.5.1",
+    "vue-property-decorator": "^9.1.2",
     "vue-template-compiler": "2.6.12",
```

Comment:
I don't think the vue-template-compiler is needed for vue 2.7.0
> You can also remove vue-template-compiler from the dependencies - it is no longer needed in 2.7.
https://blog.vuejs.org/posts/vue-2-7-naruto.html

Reply:
You're right. At the moment this is still Vue 2.6.x... but it installs and builds and tests cleanly.

I am doing a phased upgrade in this PR. My next update will require code changes (esp unit tests)!